README.md

```diff
@@ -21,17 +21,79 @@ This Client works against versions 4 to 8 of the NGINX Plus API. The table below
 | 7 | R25 |
 | 8 | R27 |
 
-## Using the Client
+## Using the client library
 
-1. Import `github.com/nginxinc/nginx-plus-go-client/client` into your go project.
-2. Use your favorite vendor tool to add this to your `/vendor` directory in your project.
+Import the library using:
+```go
+import "github.com/nginxinc/nginx-plus-go-client/client"
+```
+
+## Creating a client
+
+When creating a new ```client``` it is assumed that NGINX API is reachable. The ```client``` verifies API version of the running NGINX at the time of creating a new client object.
+
+If user wants to create a client without NGINX automatic version check it should use the ```NewDefaultNginxClient``` func to create the client.
+
+
+### Create a new ```Client``` that works with the latest NGINX API Version:
+```go
+myHTTPClient := &http.Client{
+    Timeout: 5*time.Second
+}
+c, err := NewNginxClient(myHTTPClient, "your-api-endpoint")
```

Comment:
Sometimes it's **client**.New... (see line 63 for example), sometimes it's not (as in here). Can we keep this consistent?